flocker/acceptance/test_api.py

```diff
@@ -321,11 +324,12 @@ def query(host, port):
             req = get(
                 "http://{host}:{port}".format(host=host, port=port),
                 persistent=False
-            ).addCallback(content)
+            )
+            req.addCallbacks(content, lambda _: False)
```

Comment:
You can tell that the request bit was started from the `loop_until` initial log message, so I'll just stick to logging the error.